examples/reference-entity-model/input/json/Cust1.json

```diff
@@ -0,0 +1,33 @@
+{
+    "CustomerID": 101,
```

Comment:
I like have sample files, but let's not use abbreviations - just call it "customer1.json".